modules/DMZ/data_kit/data_filler.py

```diff
@@ -2,35 +2,40 @@
 import numpy as np
 
 
-#Drops rows which have fewer than threshold observations.
-#e.g. if threshold = 5, any rows with fewer than 5 observations will be removed from pandas_data.
+# Drops rows which have fewer than threshold observations.
+# e.g. if threshold = 5, any rows with fewer than 5 observations will be removed from pandas_data.
```

Comment:
Over-documentation is a concern I'm seeing too. We do want to show at least function input/outputs for all external methods.